frontend/components/siteList/repoLastVerified.jsx

```diff
@@ -1,22 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import moment from 'moment';
```

Comment:
we don't use `moment` anywhere else in the client and this is a rather large dependency, can we accomplish this without it either by computing it another way or doing it in the backend?

Reply:
moment js addition reverted 👍🏿